docs/layouts/shortcodes/get-started/build-hook.md

```diff
@@ -0,0 +1,170 @@
+<div x-show="stack === 'python' || stack === 'nodejs' || stack === 'golang' || stack === 'java'">
+
+To build your app, you may also need to add commands to go through the build process.
+These are included in what's known as the build hook.
+
+Add something similar to the following to the end of the file you just added:
```

Comment:
```suggestion
At the end of your .platform.app.yaml file, add the following build hook example and adapt it to your needs:
```

Reply:
Agreed that we'll keep the location information in tabs for now and try to work and making that more noticeable